src/app/view/show-json-dialog/show-timestamp-as-json-dialog.component.scss

```diff
@@ -0,0 +1,129 @@
+@import "DCSA-Theme";
```

Comment:
We should simplify our CSS at some point and add it to DCSA-theme. Most of these seem to be duplicate